airflow/cli/cli_parser.py

```diff
@@ -162,6 +162,11 @@ def _check(value):
     return _check
 
 
+def string_list_type(val):
+    """Splits comma-separated list and returns the lists (strips whitespace)"""
```

Comment:
Perhaps you mean "Splits comma-separated _string_ and returns the result _in a list_"?